### PR TITLE
Experimental/sf

### PR DESCRIPTION
**Pull request by ndquocphong**

### CODE REVIEW

File: Controller/Admin/ConfigController.php [hunk 1]
```diff
@@ -10,54 +10,45 @@
...
  */
-class ConfigController extends AbstractController
+class ConfigController extends \Eccube\Controller\AbstractController
 {
+
     /**
-     * @param Application $app
-     * @param Request     $request
+     * @Route("/%eccube_admin_route%/plugin/product/review/config", name="plugin_ProductReview_config")
+     * @Template("ProductReview/Resource/template/admin/config.twig");
```
Comment (ndquocphong) [inline Controller/Admin/ConfigController.php]: You can try the shorter way (and no need `;` semicolon character)
```
@Template("@ProductReview/admin/config.twig");
```

File: Repository/ProductReviewConfigRepository.php [hunk 2]
```diff
@@ -10,14 +10,20 @@
...
+use Symfony\Bridge\Doctrine\RegistryInterface;
 
 /**
  * ProductReview Config.
  *
  * This class was generated by the Doctrine ORM. Add your own custom
  * repository methods below.
  */
-class ProductReviewConfigRepository extends EntityRepository
+class ProductReviewConfigRepository extends ServiceEntityRepository
```
Comment (ndquocphong) [inline Repository/ProductReviewConfigRepository.php]: Should extend from `\Eccube\Repository\AbstractRepository`

File: Repository/ProductReviewRepository.php [hunk 3]
```diff
@@ -10,24 +10,33 @@
...
+use Symfony\Bridge\Doctrine\RegistryInterface;
 
 /**
  * ProductReview.
  *
  * This class was generated by the Doctrine ORM. Add your own custom
  * repository methods below.
  */
-class ProductReviewRepository extends EntityRepository
+class ProductReviewRepository extends ServiceEntityRepository
```
Comment (ndquocphong) [inline Repository/ProductReviewRepository.php]: Should extend from `\Eccube\Repository\AbstractRepository`

File: Resource/config/services.php [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+// navを追加する
+$nav = (array)$container->getParameter('eccube_nav');
+foreach ($nav as $key => $value) {
+    $add = [
+        'id' => 'product_review',
+        'name' => 'レビュー管理',
+        'url' => 'plugin_admin_product_review',
+    ];
+}
```
Comment (ndquocphong) [inline Resource/config/services.php]: What does this code used for ?

File: Resource/template/default/confirm.twig [hunk 5]
```diff
@@ -13,41 +13,114 @@
...
+            }
+        }
+    </style>
 {% endblock stylesheet %}
 
+{% form_theme form 'Form/form_layout.twig' %}
+
 {% block main %}
-<h1 class="page-heading">レビューを書く</h1>
+<h1 class="page-heading">{{ ''|trans }}</h1>
```
Comment (ndquocphong) [inline Resource/template/default/confirm.twig]: Seem typo mistake here

File: Resource/template/default/error.twig [hunk 6]
```diff
@@ -12,8 +12,37 @@
...
+            font-weight: bold;
+            height: 44px;
+            border-radius: 0;
+        }
+    </style>
+{% endblock stylesheet %}
+
 {% block main %}
-    <h1 class="page-heading">レビューを書く - エラー</h1>
+    <h1 class="page-heading">{{ '{{ ''|trans }}'|trans }}</h1>
```
Comment (ndquocphong) [inline Resource/template/default/error.twig]: What does this code ?

File: Resource/template/admin/index.twig [hunk 7]
```diff
@@ -80,145 +83,194 @@ function fnChangeActionSubmit(action) {
...
-        <div class="row btn_area">
-            <div class="col-xs-8 col-xs-offset-2 col-sm-4 col-sm-offset-4 text-center">
-                <button class="btn btn-primary btn-block btn-lg" onclick="fnChangeActionSubmit('{{ url('plugin_admin_product_review') }}'); return false;">
-                    検索する <svg class="cb cb-angle-right"><use xlink:href="#cb-angle-right"></use></svg>
-                </button>
-            </div> <!-- /.col -->
+        <div class="c-subContents collapse ec-collapse" id="searchDetail">
+            <div class="row mb-2">
+                <div class="col-6 col-sm-4">
+                    <label>{{ searchForm.product_name.vars.label }}</label>
```
Comment (ndquocphong) [inline Resource/template/admin/index.twig]: Should have `trans` to translate label 
```
 searchForm.product_name.vars.label|trans
```

Comment (toannguyen-lockon) [reply Resource/template/admin/index.twig]: This code is used to get the label of the field

**Comment by hoand-vn** [comment]
fixed review

**Pull request closed without merging**